tgui/packages/tgui/interfaces/MedicalRecords.js

```diff
@@ -715,18 +665,9 @@ const MedicalRecordsNavigation = (_properties, context) => {
         >
           Virus Database
         </Tabs.Tab>
-        <Tabs.Tab
-          icon="vial"
-          selected={screen === 6}
-          onClick={() => {
-            act('screen', { screen: 6 });
-          }}
-        >
-          Virology Goals
-        </Tabs.Tab>
         <Tabs.Tab
           icon="plus-square"
-          selected={screen === 7}
+          selected={screen === 6}
           onClick={() => act('screen', { screen: 7 })}
```

Comment:
```suggestion
          onClick={() => act('screen', { screen: 6 })}
```
Does this now have to be adjusted to screen 6 when clicking on the tab?